### PR TITLE
Ft 237 token set tree

### PR DESCRIPTION
**Pull request by MightyDevKnight**

### CODE REVIEW

File: src/app/components/TokenSetSelector.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ export default function TokenSetSelector() {
     setShowNewTokenSetFields(false);
     handleNewTokenSetNameChange('');
   }, [tokens]);
-
+  console.log(allTokenSets);
```
Comment (six7) [inline src/app/components/TokenSetSelector.tsx:59]:
```suggestion
```

File: src/stories/TokenSetTree.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+<Meta 
+  title='TokenSetTree' 
+  component={TokenSetTree}
+  args={{
+    tokenSets: [
+      'core',
+      'semantic',
+      'button',
+      'icon-button',
```
Comment (six7) [inline src/stories/TokenSetTree.stories.mdx:16]: can you add nest a few of these files to become folders?

e.g. core/base, core/radii, core/colors and components/link, components/radio, ...

**Review by six7 — CHANGES_REQUESTED** [review]

File: src/app/components/TokenSetSelector.tsx (hunk 1 above)
Comment (MightyDevKnight) [reply src/app/components/TokenSetSelector.tsx:59]: Done

File: src/stories/TokenSetTree.stories.mdx (hunk 2 above)
Comment (MightyDevKnight) [reply src/stories/TokenSetTree.stories.mdx:16]: Done

**Review by six7 — APPROVED** [review]

**Pull request merged**